Source/WebCore/platform/graphics/mac/controls/ProgressBarMac.mm

```diff
@@ -67,16 +67,28 @@
 
 FloatRect ProgressBarMac::rectForBounds(const FloatRect& bounds, const ControlStyle& style) const
 {
-    int minimumProgressBarHeight = sizeForSystemFont(style).height();
-    if (bounds.height() > minimumProgressBarHeight)
+    auto isVerticalWritingMode = style.states.contains(ControlStyle::State::VerticalWritingMode);
```

Comment:
Could just be a `bool`

Reply:
Leaving this as `auto`, since that's what we generally use in new code, and it's clear from both the left and right hand sides that this is a `bool`.